docs/en/observability/get-started.asciidoc

```diff
@@ -0,0 +1,32 @@
+[chapter, role="xpack"]
+[[install-observability]]
+= Get started
+
+To use the Elastic Observability, you need {ess} for storing and searching your 
+data, and {kib} for visualizing and managing it.
+
+[float]
+[[set-up-on-cloud]]
+== Set up on Cloud
+
+To get started quickly, you can spin up a deployment of our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]. The deployment includes
+{es}, {kib}, and APM Server, and is available on AWS, GCP, and Azure.
```

Comment:
I think you can get rid of this paragraph. The content is mostly repeated in the docs-root include on line 16